src/rev_ai/apiclient.py

```diff
@@ -143,18 +156,16 @@ def submit_job_local_file(
             rush=None,
             test_mode=None,
             segments_to_transcribe=None,
-            speaker_names=None):
+            speaker_names=None,
+            notification_config=None):
         """Submit a local file for transcription.
         Note that the content type is inferred if not provided.
 
         :param filename: path to a local file on disk
         :param metadata: info to associate with the transcription job
-        :param callback_url: callback url to invoke on job completion as a
-                             webhook
-        :param skip_diarization: should Rev AI skip diaization when
-                                 transcribing this file
-        :param skip_punctuation: should Rev AI skip punctuation when
-                                 transcribing this file
+        :param callback_url: callback url to invoke on job completion as a webhook
```

Comment:
We should note that this parameter is deprecated.

https://stackoverflow.com/questions/63902170/mark-a-specific-method-or-function-parameter-as-deprecated-in-sphinx